tests/e2e/tests/devfiles/JavaVertx.spec.ts

```diff
@@ -28,19 +28,23 @@ suite(`${stack} test`, async () => {
         projectAndFileTests.waitWorkspaceReadiness(workspaceSampleName, workspaceRootFolderName);
     });
 
-    suite('Language server validation', async () => {
+    suite('Test opening file', async () => {
```

Comment:
This seems like a change in test logic not really related to the original intent of this PR...

Reply:
reverted